build.gradle

```diff
@@ -88,6 +88,16 @@ def exportedProjects = [
         ":payment-service-api"
 ]
 
+def baseAndFlowService = [
```

Comment:
Why are these changes here? They are not correct for v1 where there are three api modules